src/components/pages/Dashboard/Dashboard.js

```diff
@@ -1,5 +1,7 @@
 import React, { useEffect, useState } from 'react';
-import axiosWithAuth from '../../../utils/axiosWithAuth';
+import useAxiosWithAuth0 from '../../../hooks/useAxiosWithAuth0';
+import { Statistic, Row, Col, Table } from 'antd';
```

Comment:
Any reason why there are errors saying Statistic, Row, and Col are defined but not used?

Reply:
It looks like this is because they were going to be used here in the Dashboard for rendering out data from the DS team, but since this page hasn't been fully implemented yet and we've since pivoted to using a real visualization library for the same, they're going unused.

The imports for these unused Ant D components could go, but let's handle this in a separate cleanup PR.